projects/ziti-console-lib/src/lib/features/overrides/overrides.component.scss

```diff
@@ -32,4 +32,43 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+::ng-deep .override-list-container {
+  .p-element.p-icon-wrapper path {
+    stroke: var(--formSubGroup) !important;
+    fill: var(--formSubGroup) !important;
+  }
+  .p-dropdown {
+    font-weight: 400;
+    font-family: 'Open Sans', Arial, sans-serif;
+    height: 50px;
+    width: 100%;
+    border-color: var(--stroke);
+    border-style: solid;
+    border-width: 2px;
```

Comment:
are these values always bespoke per component? Is there "a good reason" not to standardize all this an use a css variable? seems like these are the sorts of things that would be good to keep consistent across the whole ui? we can discuss offline